### ISSUE TITLE
TouchCursor doesn't work with the Everything search tool

### ISSUE THREAD

**Issue by asampal** [martin-stone/touchcursor#2]
TouchCursor works with most things, but unfortunately doesn't with [Everything](http://www.voidtools.com), my favourite file search tool. Is there a specific API that must be used so that TouchCursor can intercept keys? If so, I could raise an issue with the developer or possibly try and update the TouchCursor source to support additional APIs.

**Comment by martin-stone** [comment]
"Everything" seems to ask for Administrator rights. TouchCursor often doesn't work in programs running as Administrator. It might be possible to get it working if TouchCursor can also be launched in Administrator mode. I haven't tried this yet.

If I say "no" to the Administrator prompts in Everything, TouchCursor works with it. (It doesn't index my drive though). I note that "Everything" has an option to run as a service instead of using Administrator. Have you tried this? (I haven't looked into that myself yet.)

**Comment by asampal** [comment]
Hmm, you're right it does work when not run as Administrator. I've never ran Everything as a service before - always "installed" it as a single file portable app. Anyhow, it's easy enough to change the settings of "run as Administrator" or "run as service" from Everything's Options menu (under the General node) so I just unchecked the Administrator option and switched to service - it works! Thanks for the heads up on that.

I'm curious, though, why it wouldn't work when apps are using the Admin user since there have been other odd cases that didn't work with TouchCursor. Have you looked into it yourself in any depth?

**Comment by martin-stone** [comment]
It seems reasonable that TouchCursor wouldn't intercept keys for the admin user when it is installed and running for some other user. There are surely good security reasons for that. 

Some things that I've noticed that don't work: Editing environment variables in the Advanced System Settings, and entering username and password at a Windows log-in screen. 

I have tried installing TouchCursor as admin and that didn't make it start working in those situations. Maybe more work needs doing to get it to actually _run_ as admin. For me, the problem areas aren't common enough for me to have tried any harder.
